briefcase/app.py

```diff
@@ -159,10 +164,6 @@ def app_dir(self):
     def app_packages_dir(self):
         return os.path.join(os.getcwd(), self.resource_dir, 'app_packages')
 
-    @property
-    def version(self):
```

Comment:
this is definitely used by some of the templates, at the very least the Django one. I don't think this is safe to remove?

Reply:
It was replaced with the self.version variable set above which would/should behave the same, other than technically being writable and not necessarily created early enough.